### PR TITLE
Replace `route` with `mydumper.files` due to `route` feature not GA yet

### PR DESCRIPTION
**Pull request by sunzhaoyang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: migrate-large-mysql-shards-to-tidb.md [hunk 1]
```diff
@@ -138,17 +142,15 @@ tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my
...
-    target-schema = "my_db"
-    target-table = "table5"
-
-    [[routes]]
-    schema-pattern = "my_db2"
-    table-pattern = "table[3-4]"
-    target-schema = "my_db"
-    target-table = "table5"
+    [[mydumper.files]]
+    pattern = '(?i)^[^/]*/testdm\.t[1-2].*\.sql$'
```
Comment (gozssky) [inline migrate-large-mysql-shards-to-tidb.md:146]: pattern 和前面 dumpling 导出的文件路径好像不太对得上

Comment (gozssky) [reply migrate-large-mysql-shards-to-tidb.md:146]: testdm?

File: migrate-large-mysql-shards-to-tidb.md [hunk 2]
```diff
@@ -115,6 +115,10 @@ tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my
 
 关于断点续传的更多信息，请参考 [TiDB Lightning 断点续传](/tidb-lightning/tidb-lightning-checkpoints.md)。
 
+### 在下游 创建schema
+
+在`${data-path}`中使用`my_db1-schema-create.sql`文件，编辑后手工在下游创建`mydb.table5`。
```
Comment (shichun-0415) [inline migrate-large-mysql-shards-to-tidb.md:120]:
```suggestion
在 `${data-path}` 中使用 `my_db1-schema-create.sql` 文件，编辑后手工在下游创建 `mydb.table5`。
```

**Review by shichun-0415 — APPROVED** [review]

File: migrate-large-mysql-shards-to-tidb.md [hunk 3]
```diff
@@ -115,6 +115,10 @@ tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my
 
 关于断点续传的更多信息，请参考 [TiDB Lightning 断点续传](/tidb-lightning/tidb-lightning-checkpoints.md)。
 
+### 在下游 创建schema
```
Comment (shichun-0415) [inline migrate-large-mysql-shards-to-tidb.md:118]:
```suggestion
### 在下游创建schema
```

File: migrate-large-mysql-shards-to-tidb.md [hunk 4]
```diff
@@ -115,6 +115,10 @@ tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my
 
 关于断点续传的更多信息，请参考 [TiDB Lightning 断点续传](/tidb-lightning/tidb-lightning-checkpoints.md)。
 
+### 在下游创建schema
```
Comment (shichun-0415) [inline migrate-large-mysql-shards-to-tidb.md:118]:
```suggestion
### 在下游创建 schema
```

File: migrate-large-mysql-shards-to-tidb.md [hunk 5]
```diff
@@ -115,6 +115,10 @@ tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my
 
 关于断点续传的更多信息，请参考 [TiDB Lightning 断点续传](/tidb-lightning/tidb-lightning-checkpoints.md)。
 
+### 在下游创建 schema
+
+在 `${data-path}` 中使用 `my_db1-schema-create.sql` 文件，编辑后手工在下游创建 `mydb.table5`。
```
Comment (shichun-0415) [inline migrate-large-mysql-shards-to-tidb.md:120]: Maybe better to provide a copyable command for this operation.

Comment (sunzhaoyang) [inline migrate-large-mysql-shards-to-tidb.md:120]:
```suggestion
在 `${data-path}` 中使用 `my_db1-schema-create.sql` 文件，编辑后手工在下游创建 `mydb.table5`。

{{< copyable "shell-regular" >}}

```
vim ${data-path}/my_db1/my_db1-schema-create.sql
```
```

File: migrate-large-mysql-shards-to-tidb.md [hunk 6]
```diff
@@ -115,6 +115,16 @@ tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my
...
+
+{{< copyable "shell-regular" >}}
+
+```
```
Comment (shichun-0415) [inline migrate-large-mysql-shards-to-tidb.md:124]:
```suggestion
```shell
```

**Review by shichun-0415 — APPROVED** [review]

File: migrate-large-mysql-shards-to-tidb.md [hunk 7]
```diff
@@ -115,6 +115,16 @@ tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my
...
 
+### 在下游创建 schema
+
+在 `${data-path}` 中使用 `my_db1-schema-create.sql` 文件，编辑后手工在下游创建 `mydb.table5`。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+vim ${data-path}/my_db1/my_db1-schema-create.sql
+```
```
Comment (gozssky) [inline migrate-large-mysql-shards-to-tidb.md:126]: mydumper.files 没有匹配这个 schema 文件

File: migrate-large-mysql-shards-to-tidb.md [hunk 8]
```diff
@@ -138,17 +148,15 @@ tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my
...
-    target-schema = "my_db"
-    target-table = "table5"
-
-    [[routes]]
-    schema-pattern = "my_db2"
-    table-pattern = "table[3-4]"
-    target-schema = "my_db"
-    target-table = "table5"
+    [[mydumper.files]]
+    pattern = '(?i)^[^/]*/my_db1\.t[1-2].*\.sql$'
```
Comment (gozssky) [inline migrate-large-mysql-shards-to-tidb.md:152]:
```suggestion
    pattern = '(^|/)my_db1\.table[1-2].*\.sql$'
```

File: migrate-large-mysql-shards-to-tidb.md [hunk 9]
```diff
@@ -138,17 +148,15 @@ tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my
...
+    schema = "my_db"
+    table = "table5"
+
+    [[mydumper.files]]
+    pattern = '(?i)^[^/]*/my_db2\.t[3-4].*\.sql$'
```
Comment (gozssky) [inline migrate-large-mysql-shards-to-tidb.md:157]:
```suggestion
    pattern = '(^|/)my_db1\.table[1-2].*\.sql$'
```

File: migrate-large-mysql-shards-to-tidb.md [hunk 10]
```diff
@@ -138,17 +148,15 @@ tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my
...
-
-    [[routes]]
-    schema-pattern = "my_db2"
-    table-pattern = "table[3-4]"
-    target-schema = "my_db"
-    target-table = "table5"
+    [[mydumper.files]]
+    pattern = '(?i)^[^/]*/my_db1\.t[1-2].*\.sql$'
+    schema = "my_db"
+    table = "table5"
```
Comment (gozssky) [inline migrate-large-mysql-shards-to-tidb.md:154]:
```suggestion
    table = "table5"
    type = "sql"
```

File: migrate-large-mysql-shards-to-tidb.md [hunk 11]
```diff
@@ -138,17 +148,15 @@ tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my
...
+
+    [[mydumper.files]]
+    pattern = '(?i)^[^/]*/my_db2\.t[3-4].*\.sql$'
+    schema = "my_db"
+    table = "table5"
```
Comment (gozssky) [inline migrate-large-mysql-shards-to-tidb.md:159]:
```suggestion
    table = "table5"
    type = "sql"
```

File: migrate-large-mysql-shards-to-tidb.md (hunk 7 above)
Comment (sunzhaoyang) [reply migrate-large-mysql-shards-to-tidb.md:126]: 嗯，我想着写匹配规则太麻烦了，上面的描述是手动跑下，建出来算了...

File: migrate-large-mysql-shards-to-tidb.md [hunk 12]
```diff
@@ -115,6 +115,16 @@ tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my
 
 关于断点续传的更多信息，请参考 [TiDB Lightning 断点续传](/tidb-lightning/tidb-lightning-checkpoints.md)。
 
+### 在下游创建 schema
+
+在 `${data-path}` 中使用 `my_db1-schema-create.sql` 文件，编辑后手工在下游创建 `mydb.table5`。
```
Comment (sunzhaoyang) [inline migrate-large-mysql-shards-to-tidb.md:120]:
```suggestion
在 `${data-path}` 中使用 `my_db1-schema-create.sql` 文件，编辑后手工在下游执行以创建 `mydb.table5`。
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: migrate-large-mysql-shards-to-tidb.md (hunk 7 above)
Comment (gozssky) [reply migrate-large-mysql-shards-to-tidb.md:126]: 手动创建的话，就不用创建这个文件了，反而可能会引起用户困扰。

File: migrate-large-mysql-shards-to-tidb.md [hunk 13]
```diff
@@ -115,6 +115,16 @@ tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my
 
 关于断点续传的更多信息，请参考 [TiDB Lightning 断点续传](/tidb-lightning/tidb-lightning-checkpoints.md)。
 
+### 在下游创建 schema
+
+在 `${data-path}` 中使用 `my_db1-schema-create.sql` 文件，编辑后手工在下游执行以创建 `mydb.table5`。
```
Comment (shichun-0415) [inline migrate-large-mysql-shards-to-tidb.md:120]: 这里有个疑问，”编辑后“具体要编辑成什么样子？

Comment (sunzhaoyang) [inline migrate-large-mysql-shards-to-tidb.md:120]:
```suggestion
手工在下游创建`mydb.table5` ，例如编辑`my_db1-schema-create.sql` 文件，将表名改为`mydb.table5`后在下游执行。
```

File: migrate-large-mysql-shards-to-tidb.md (hunk 7 above)
Comment (sunzhaoyang) [reply migrate-large-mysql-shards-to-tidb.md:126]: 嗯。这个是 dumpling 导出的来的文件，用户改改直接跑就行，并没有新建其实。

**Review by gozssky — APPROVED** [review]

**Pull request merged**